worlds/dlcquest/Items.py

```diff
@@ -11,6 +11,8 @@
 
 class DLCQuestItem(Item):
     game: str = "DLCQuest"
+    coins: int = 0
```

Comment:
in 2066, I've tried to add consistency in defining the base campaign stuff either "dlcquest" or simply "basic", as opposed to not giving it any name. This is for it to be easier to distinguish "coin" (dlc quest coin) vs "coin" (any coin). I would recommend following this pattern here as well